sdk/storage/storage-blob/review/storage-blob.api.md

```diff
@@ -511,6 +517,7 @@ export interface BlobDownloadOptionalParams extends coreHttp.RequestOptionsBase
     requestId?: string;
     snapshot?: string;
     timeoutInSeconds?: number;
+    versionId?: string;
```

Comment:
Do we still want this here? @jeremymeng suggested the `snapshot` optional param was only there since we inherited from v10. I am ok if `versionId` must only be specified via `withVersion`.

Reply:
We already removed the `versionId` in the `BlobDownloadOptions`. This `BlobDownloadOptionalParams` is the one from the protocol layer so it's including everything.